Automattic/wp-calypso
# Site Editor: Track Global Styles button click in header

#### Changes proposed in this Pull Request

* Track Global Styles button in header, event name: `wpcom_block_editor_global_styles_tab_selected`

#### Testing instructions



1. Follow steps to enable tracking debugging at PCYsg-nrf-p2
2. Open Site Editor
3. Click on the Global Styles button in the header (the one with "aA" icon)
4. Make sure `wpcom_block_editor_global_styles_tab_selected` event is fired with `{ tab: 'root', open: true }`
5. Click on Block Type tab
6. Make sure `wpcom_block_editor_global_styles_tab_selected` event is fired with `{ tab: 'block-type', open: true }`
7. Click on Root tab
8. Make sure `wpcom_block_editor_global_styles_tab_selected` event is fired with `{ tab: 'root', open: true }`
9. Open block settings (click on cog next to the "aA" icon)
10. Make sure `wpcom_block_editor_global_styles_tab_selected` event is fired with `{ tab: 'root', open: false }`
11. Click on the Global Styles button in the header (the one with "aA" icon)
12. Make sure `wpcom_block_editor_global_styles_tab_selected` event is fired with `{ tab: 'root', open: true }`
13. Click on Block Type tab
14. Make sure `wpcom_block_editor_global_styles_tab_selected` event is fired with `{ tab: 'block-type', open: true }`
15. Open block settings (click on cog next to the "aA" icon)
16. Make sure `wpcom_block_editor_global_styles_tab_selected` event is fired with `{ tab: 'block-type', open: false }`

And make sure e2e tests are passing:
```bash
cd test/e2e
./node_modules/.bin/mocha specs/specs-wpcom/wp-calypso-gutenberg-site-editor-tracking-spec.js
```



Related to https://github.com/Automattic/wp-calypso/issues/53410
